app/controllers/concerns/foreman/controller/parameters/external_usergroup.rb

```diff
@@ -0,0 +1,17 @@
+module Foreman::Controller::Parameters::ExternalUsergroup
+  extend ActiveSupport::Concern
+
+  class_methods do
+    def external_usergroup_params_filter
+      Foreman::ParameterFilter.new(::ExternalUsergroup).tap do |filter|
+        filter.permit :usergroup, :usergroup_id, :usergroup_name
+        filter.permit_by_context :name, :auth_source_id, :auth_source_name, :nested => true
+        filter.permit_by_context :id, :_destroy, :ui => false, :api => false, :nested => true
```

Comment:
Only a subjective opinion, for better git merging, I'd prefer one item per line. It's a chance to set the standards. I tend to prefer practical approach to appearance. Just for comparison, this one would look like:

```ruby
filter.permit
  :usergroup,
  :usergroup_id,
  :usergroup_name
filter.permit_by_context
  :name,
  :auth_source_id,
  :auth_source_name,
  :id,
  :_destroy,
  :ui => false,
  :api => false,
  :nested => true
```

I understand I am part of minority in this case. :-)